### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by Ba1-Tazar** [cc-tweaked/CC-Tweaked#1782] [Q&A]
How do i copy paste the code from VS studio .lua file to minecraft ?

**Comment by SquidDev** [comment]
If you're on a single player world, you can copy the file directly to the computer's directory. That can be found at .minecraft/saves/<world>/computercraft/computer/<id>, where <world> is the save name of the world, and <id> is the ID of the computer, found with the id command.

Otherwise, you should be able to drag and drop the file from your file browser onto the computer's GUI to upload it.

**Reply by Lupus590** [reply]
You can also upload it to file sharing services.
There are some that specialise in text files (a lua file is just a text file with a funny name), [Pastebin](https://pastebin.com/) is quite popular and CC has a program for downloading from it.
